Prog/Tabulate householder population for projection.sas

```diff
@@ -0,0 +1,247 @@
+/**************************************************************************
+ Program:  tabulate householder population for projection.sas
+ Library:  NCHsg
+ Project:  NeighborhoodInfo DC
+ Author:   Yipeng Su
+ Created:  7/16/19
+ Version:  SAS 9.2
+ Environment:  Local Windows session (desktop)
+ 
+ Description:  Produce detailed popualtion by age group, race ethnicity and jurisciation from 2008-2017
+ ACS IPUMS data for the NC region:
+
+
+ Modifications: 01/16/19 LH Update incomecat for capped 80% of AMI. Add date for output. 
+			    02/13/19 LH Change relate for 2 serials to better reflect household structure. 
+**************************************************************************/
+
+%include "L:\SAS\Inc\StdLocal.sas";
+
+** Define libraries **;
+%DCData_lib( NCHsg)
+%DCData_lib( Ipums)
+
+%let date=07192019;
+
+proc format;
+
+	value agegroupnew
+	.n= 'Not available'
+	1='under 25 years old'
+	2= '25-45 years old'
+	3= '45-65 years old'
+	4='65+ years old';
+  value race
+   .n = 'Not available'
+    1 = 'White'
+    2 = 'Black'
+    3 = "All Other ";
+  value hispan
+     .n = 'Not available'
+    0 = 'Not Hispanic'
+    1 = 'Hispanic';
+ value racenew
+   .n = 'Not available'
+    1 = 'White non-Hispanic'
+    2 = 'Black non-Hispanic'
+    3 = "Hispanic "
+	4 = "All other non-Hispanic ";
+  value agegroup
+     .n = 'Not available'
+    1= "0-5 years old"
+	2= "5-9 years old"
+	3= "10-14 years old "
+	4="15-19 years old"
+	5="20-24 years old "
+	6="25-29 years old"
+	7="30-34 years old"
+	8="35-39 years old"
+	9="40-44 years old"
+    10="45-49 years old"
+    11= "50-54 years old"
+    12= "55-59 years old"
+    13= "60-64 years old"
+    14= "65-69 years old"
+    15= "70-74 years old"
+    16= "75-79 years old" 
+    17 = "80-84 years old"
+    18= "85+ years old";
+run;
+
+
+%macro householdinfo(year);
+
+
+	data Household_&year. ;
+		set Ipums.Acs_&year._NC;
+
+		%assign_NCcounty;
+		%assign_NCcounty;
+		county_char = put(county, 5.);
+	run;
+
+	data Inc_&year. ;
+	set NCHsg.IncomeLimits_&year. (where= (State=37));
+        county_new = put(County,z3.);
+        state2= put(State, z2.);
+        county_char= state2||county_new;
+	run;
+
+	proc sort data= Household_&year. ;
+	by county_char;
+	run;
+
+	proc sort data= Inc_&year. ;
+	by county_char;
+	run;
+
+	data Household_&year._2 ;
+	merge Household_&year. Inc_&year.;
+	by county_char ;
+	run;
+
+	data Householddetail_&year.;
+		set Household_&year._2;
+		keep race hispan age hhincome hhincome_a pernum relate gq county hhwt perwt year serial numprec race1 agegroup hud_inc totpop_&year. l50_1- l50_8 l80_1- l80_8  median&year.   ;
+
+		 %Hud_inc_NCHsg( hhinc=hhincome_a, hhsize=numprec )
```

Comment:
do we actually need the income in this program at all? i'd remove all pieces that aren't necessary to this program. (also you'd want to adjust for inflation..)